plugins/SitesManager/lang/en.json

```diff
@@ -202,6 +202,10 @@
         "SiteWithoutDataVueFollowStepCompleted": "%1$sCongratulations!%2$s You have successfully installed the Matomo Analytics tracking code via the \"%3$s\" npm package. To verify that hits are being tracked, visit your website and check that this data is visible in your Matomo instance.",
         "SiteWithoutDataVueFollowStepNote1": " Now you can access Matomo in your components through %1$s.",
         "SiteWithoutDataVueFollowStepNote2": "If you followed these steps and have no data coming into Matomo, you can try the %1$sMatomo Tag Manager%2$s instead (use this link for %3$sSingle Page Applications%4$s).",
-        "VueDetectedEmail": "Vue.js was detected on your website. Did you know you can use the \"%1$s\" npm package to track the website data? Learn more here: %2$s"
+        "VueDetectedEmail": "Vue.js was detected on your website. Did you know you can use the \"%1$s\" npm package to track the website data? Learn more here: %2$s",
+        "MergedNotificationLine1": "%1$s were detected on your website.",
+        "MergedNotificationLine2": "Learn more about configuring Matomo to work with these by following these guides: %1$s",
+        "ConsentManagerConnected": "We’ve detected that %1$s is correctly configured to work with Matomo on your site.",
+        "Ampersand": "and"
```

Comment:
Please use `General_And` instead, as that one should already be translated to various languages.